extensions/devservices/keycloak/runtime/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-devservices-keycloak-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-devservices-keycloak</artifactId>
+    <name>Quarkus - DevServices - Keycloak</name>
+    <dependencies>
+        <dependency>
+          <groupId>io.quarkus</groupId>
+          <artifactId>quarkus-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
```

Comment:
We can't be bringing extra dependencies into runtime to make devservices work. All this sort of stuff needs to be in the deployment module.

Reply:
@stuartwdouglas Hi Stuart, i followed an approach taken by the cache extension where the template actions are backed up by the recorder code - but in this case the vert.x and keycloak clients need to acqure/post the token - can be done outsode of the recorder ?